lib/active_merchant/billing/gateways/paypal_express_common.rb

```diff
@@ -16,13 +16,11 @@ def redirect_url
         test? ? test_redirect_url : live_redirect_url
       end
 
-      def redirect_url_for(token, options = {})
+      def base_redirect_url_for(token, options: {}, cmd: nil)
         options = {:review => true, :mobile => false}.update(options)
```

Comment:
Why have a combination of named parameter and an options hash? Let's change it all to named parameters. 

Also, `mobile => false` has no more sense anymore in this context. 

I'm not fond of classes or methods names `base_*`. WDYT of naming it `build_redirect_url(`